openshift/ci-operator/build-image/setup-logging-for-e2e.sh

```diff
@@ -54,6 +54,13 @@ else
     testroot=/go/src/github.com/openshift/origin-aggregated-logging
 fi
 make undeploy > /dev/null 2>&1 || :
+# use less es memory and cpu
+cp hack/cr.yaml hack/cr.yaml.save
+python -c 'import sys,yaml
+hsh = yaml.load(open(sys.argv[1]))
+hsh["spec"]["logStore"]["elasticsearch"]["resources"] = {"limits":{"cpu":"2","memory":"4Gi"}}
```

Comment:
couldn't we get away with .5 or 1 for the CPU? 2 seems high still for the CI machines

Reply:
old CI used 400m for es cpu - let's try 500m 
https://github.com/openshift/aos-cd-jobs/blob/master/sjb/inventory/group_vars/OSEv3/logging.yml#L15